log/sequencer.go

```diff
@@ -325,22 +322,21 @@ func (s Sequencer) IntegrateBatch(ctx context.Context, logID int64, limit int, g
 		return 0, s.SignRoot(ctx, logID)
 	}
 
-	st := &logSequencingTask{
+	var st sequencingTask = logSequencingTask{
```

Comment:
`sequencingTask := logSequencingTask{...}`

(but `st :=` was fine too - golang style prefers short names over verbose ones where possible)

Reply:
This is to use sequencincTask interface explicitly. Otherwise the linter complains about unused type.

Reply:
The name is still 'st'.

Reply:
Ah, oke!